ATeam-FantasyGame/ATeam-FantasyGame.cpp

```diff
@@ -1,11 +1,20 @@
 // ATeam-FantasyGame.cpp : This file contains the 'main' function. Program execution begins and ends there.
-//
+// test test
 
 #include <iostream>
+#include "Backstory.h"
```

Comment:
Should move Backstory to its own singleton class with a BackStory.cpp and BackStory.h file.